README.md

```diff
@@ -1,2 +1,146 @@
-# cataloger
-A catalog label assignment and management system for items stored digitally, using PostgreSQL and Ruby
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Cataloger ](#-cataloger-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Video Demo ](#-video-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Cataloger <a name="about-project"></a>
+
+Cataloger: In this project, we created a console app that will help user to keep a record of different types of things you own: books, music albums, and games. The data will be stored in JSON files. It also includes a schema file with tables of the classes in the app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Languages: Ruby, SQL
+- Frameworks: Rspec
+- Technologies Used: GIT, GITHUB, LINTERS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **Add books, music albums and games**
+- **List all books, labels, music albums, genres, games & authors**
+- **Data preserved in json file**
+- **Schema file with tables of the classes in the app**
+
+<!-- video DEMO -->
+
+## 🚀 Video Demo <a name="video-demo"></a>
+
+- [Video Presentation](https://drive.google.com/file/d/1qlVujgklgt2I1tAiO1a93D11yZplsaB0/view?usp=sharing)
```

Comment:
Your presentation is a bit long and it is over the maximum time allowed

- [ ] Kindly record a new video and keep it under the maximum length of 5 mins